configurationapi/.openapi-generator-ignore

```diff
@@ -36,3 +36,7 @@ git_push.sh
 # And so that we use the top-level go.mod and go.sum files in the repository root
 go.mod
 go.sum
+
+client.go
```

Comment:
Do client and configuration need to be ignored? Is there anything hand-written in them? And as far as I know we don't need to specify files that the generator doesn't create itself here, so you can probably leave off oauth2.go.

Reply:
Will remove them. It does make sense if they're generated